### PR TITLE
New version: Ryujinx.Ryujinx.Ava version 1.1.1110

### PR DESCRIPTION
**Pull request by Exorcism0666**
###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/microsoft/winget-pkgs/pull/133980)

### CODE REVIEW

**Comment by Exorcism0666** [comment]
## For moderators

This account is automated by Github Actions and the source code was created by CoolPlayLin. If you have any questions about any pull request, don't hesitate to ping @Exorcism0666, I'll get a notification.
>[!important]
>Please carefully review these Pull Request before merging. If it is a Pull Request for removing incorrect content and the URLs are issue, free upon manual checking, please close this Pull Request directly. (It is best to inform Exorcism0666 of the closure.)

良い一日をお過ごしください！

File: manifests/r/Ryujinx/Ryujinx/Ava/1.1.1110/Ryujinx.Ryujinx.Ava.locale.en-US.yaml [hunk 1]
```diff
@@ -0,0 +1,18 @@
+# Created with Komac v1.11.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.5.0.schema.json
+
+PackageIdentifier: Ryujinx.Ryujinx.Ava
+PackageVersion: 1.1.1110
+PackageLocale: en-US
+Publisher: Ryujinx
+PublisherUrl: https://ryujinx.org
```
Comment (agatemosu) [inline manifests/r/Ryujinx/Ryujinx/Ava/1.1.1110/Ryujinx.Ryujinx.Ava.locale.en-US.yaml:8]:
```suggestion
PublisherUrl: https://ryujinx.org/
```

**Review by agatemosu — COMMENTED** [review]
your links are missing a /

**Comment by stephengillie** [comment]
Automatic Validation ended with:
>  Exception during executable launch operation System.InvalidOperationException: No process is associated with this object.

(Automated response - build 574.)

**Comment by Exorcism0666** [comment]
aaaaaaaaa

**Review by stephengillie — APPROVED** [review]

**Pull request merged**